apps/arm/src/main.cpp

```diff
@@ -9,320 +10,476 @@
 #include "TurntableConfig.h"
 #include "WristConfig.h"
 
-/*** ARM COMMAND HANDLER FUNCTIONS ***/
-/*************************************/
-
-static mbed_error_status_t setControlMode(CANMsg &msg) {
-  HWBRIDGE::CONTROL::Mode controlMode;
-  msg.getPayload(controlMode);
-  bool success;
-
-  switch (msg.getID()) {
-    case HWBRIDGE::CANID::SET_TURNTABLE_CONTROL_MODE:
-      success = Turntable::manager.switchControlMode(controlMode);
-      break;
-    case HWBRIDGE::CANID::SET_SHOULDER_CONTROL_MODE:
-      success = Shoulder::manager.switchControlMode(controlMode);
-      break;
-    case HWBRIDGE::CANID::SET_ELBOW_CONTROL_MODE:
-      success = Elbow::manager.switchControlMode(controlMode);
-      break;
-    case HWBRIDGE::CANID::SET_LEFT_WRIST_CONTROL_MODE:
-      success = Wrist::leftManager.switchControlMode(controlMode);
-      break;
-    case HWBRIDGE::CANID::SET_RIGHT_WRIST_CONTROL_MODE:
-      success = Wrist::rightManager.switchControlMode(controlMode);
-      break;
-    case HWBRIDGE::CANID::SET_CLAW_CONTROL_MODE:
-      success = Claw::manager.switchControlMode(controlMode);
-      break;
-    default:
-      return MBED_ERROR_INVALID_ARGUMENT;
-  }
-  return success ? MBED_SUCCESS : MBED_ERROR_CODE_FAILED_OPERATION;
+CANInterface can(CANConfig::config);
+
+// Convert degrees to radians
+static inline float DEG_TO_RAD(float deg) {
+  return deg * M_PI / 180.0f;
 }
```

Comment:
These DEG_TO_RAD and RAD_TO_DEG functions will be removed once the units library has been integrated by @upadhyaydhruv